### PR TITLE
Eliminate long maven build from the setup

### PR DESCRIPTION
**Pull request by merks**
- Ensure that the two projects referenced as dependency by org.eclipse.mylyn.wikitext.maven have an m2e nature and build so that the dependencies resolve to those workspace projects.

https://github.com/eclipse-mylyn/org.eclipse.mylyn/issues/552

### CODE REVIEW

**Comment by merks** [comment]
@ruspl-afed 

Maybe there's a reason to avoid doing a commit during the current post-rc2 period?

**Comment by ruspl-afed** [comment]
TL; DR;  please proceed

Speaking from the Change Management perspective we have a release tag https://github.com/eclipse-mylyn/org.eclipse.mylyn/releases/tag/R_4_3_0 and we can create a branch to hotfix 4.3.0 is something will go really wrong.

Speaking from the Release Management perspective we don't have resources to do testing for such a notable source base. We have only a few part time enthusiast that can sporadically fix something. We fully rely on automated tests we have. And the recent degradation of testing infra is a serious risk for us.

I tried to draw attention of Eclipse IDE WG to this problem [here](https://www.eclipse.org/lists/eclipse-ide-wg/msg00354.html) but without too much success. 
Also, during the recent Eclipse IDE - Developers community call,  I asked presenters from the one of Eclipse IDE WG Platinum members if they plan to contribute to Eclipse Mylyn since they use it. The answer was negative.

Well, if vendors are not interested to help Eclipse Mylyn, we can only continue our usual project reanimation protocols as we do already for a several releases. 

Thank you for reading.

**Comment by merks** [comment]
The story for Mylyn is not so different from that for BIRT, where there are a large number of people who want stuff and use stuff, from very, very large companies with lots of money.  But no one is wiling to do much of anything, except maybe complain.  😱 

The whining about wanting BIRT published to Maven central is particularly comical.

The story is pretty much the same for a great many projects...
